lib/checkPullRequestLabels.js

```diff
@@ -210,6 +213,26 @@ module.exports.checkChangelogLabel = async function (context) {
       });
       await context.github.issues.createComment(failedCommentParams);
     }
+    else if (!hasChangesBeingRequested) {
+      const author = pullRequest.user.login;
+      const allAssignees = pullRequest.assignees.map((assignee) => assignee.login);
+      if (allAssignees.includes(author)) {
```

Comment:
This can be refactored to check if the author is assigned in just one iteration
```
const authorIsAssigned = pullRequest.assignees.some(assignee => assignee.login === author)
if (authorIsAssigned) {
```

Reply:
Done.